_news-and-blog/2023-10-17-solutions-by-families-for-families.md

```diff
@@ -0,0 +1,250 @@
+team, comprised of researchers, designers, evaluators, technical experts, and policy professionals. We created several
+composite profiles from these stories that represent the challenges people face.
+
+Kailee\* is a first-time mom. When she discovered she was pregnant, she had mixed feelings, both excited and nervous,
+about embarking on this journey as a single mother who had just lost her job.
+
+During one of her first prenatal appointments, Kailee received a stack of pamphlets about different government benefit
+programs. However, she had felt ill all day, and the information overwhelmed her, so she put the information away in her bag to read when she could pay closer attention. But after leaving the doctor’s appointment she was engulfed with her other responsibilities and forgot about the benefit program pamphlets.
+
+Several months after giving birth, Kailee received an unexpected medical bill, even though she was enrolled in Medicaid. Panicked, she reached out to her church for advice. Her pastor shared that Kailee might be eligible for federally funded programs such as:
+
+- The Special Supplemental Nutrition Program for Women, Infants, and Children ([WIC](https://www.fns.usda.gov/wic))
+- The Supplemental Nutrition Assistance Program ([SNAP](https://www.fns.usda.gov/snap/supplemental-nutrition-assistance-program))
+- Temporary Assistance for Needy Families ([TANF](https://www.acf.hhs.gov/ofa/programs/temporary-assistance-needy-families-tanf))
+
+She called each office but was frustrated with the wait times or confusing phone systems. Instead, she figured going in
+person would be easier, but visiting each office required her to take several bus rides with her newborn and hours of
+sitting in a waiting room.
+
+It was a stressful situation for Kailee, and she did her best to keep her newborn comfortable during these extended
+periods. This process spanned several days, and the cost of transportation cut into her grocery budget for the week.
+
+Finally, after collecting all the required documents and providing nearly identical information in three separate
+applications, she waited to find out if she’d been approved.
+
+Weeks passed without an update, and Kailee struggled to cover the cost of baby essentials, such as diapers and a
+stroller. Stretched thin, she reached out to a moms’ mutual aid group for diapers and formula to hold her over as she
+waited for her benefits to kick in.
+
+This story was typical of many mothers and caregivers who shared their experiences with us.
+
+_\*Kailee’s name is fictional to protect her privacy._
+
+<blockquote class="pullquote" markdown="1">
+Solutions are never implemented and change never comes. There’s inquiries and roundtables, but is it really just for information-sake or is it because we’re really going to do something in the next term?
+<footer>– Community Health Worker in South Carolina who supports new moms</footer>
```

Comment:
quote attribution doesn't show up in reading mode. Not sure if this matters.

Reply:
Interesting! I'm getting inconsistent results trying to replicate/resolve this, but looks like it might have to do with the tags nested within <blockquote>. Made a bug fix issue for it here: https://github.com/usds/website-management/issues/152